apps/frontend/src/pages/manage-routes/libs/components/action-cell/action-cell.tsx

```diff
@@ -0,0 +1,49 @@
+import { memo } from "react";
+
+import { Icon } from "~/libs/components/components.js";
+import { useCallback } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type ActionCellProperties = {
+	id: number;
+	onDelete: (id: number) => void;
+	onEdit: (id: number) => void;
+};
+
+const ActionCell = memo(({ id, onDelete, onEdit }: ActionCellProperties) => {
+	const handleEditClick = useCallback((): void => {
+		onEdit(id);
+	}, [id, onEdit]);
+
+	const handleDeleteClick = useCallback(() => {
```

Comment:
just a convention how we do this in the app
```suggestion
	const handleEdit = useCallback((): void => {
		onEdit(id);
	}, [id, onEdit]);

	const handleDelete = useCallback(() => {
```